gradle/verification-metadata.xml

```diff
@@ -2948,114 +2948,59 @@
             <sha256 value="3902794d36d9b81da1b7e697f21ed04ccae276cc116eecc640a4cd0fff2691f2" origin="Generated by Gradle"/>
          </artifact>
       </component>
-      <component group="org.apache.tika" name="tika-core" version="2.7.0">
-         <artifact name="tika-core-2.7.0.jar">
-            <sha256 value="2603961edec8bcee014d2c360e5a9a0cdbf10aa8e6e95cf67abe190c837bb054" origin="Generated by Gradle"/>
+      <component group="org.apache.tika" name="tika-core" version="2.9.2">
```

Comment:
this update is the result of manually deleting the org.apache.tika dependencies here then running `./gradlew --write-verification-metadata sha256 precommit`